unlock-app/src/components/creator/members/GrantKeysDrawer.tsx

```diff
@@ -212,6 +239,26 @@ const GrantKeyForm = ({ onGranted, lock }: GrantKeyFormProps) => {
         </div>
       </div>
 
+      <div className="flex flex-wrap mb-6 -mx-3">
+        <div className="w-full px-3">
+          <Label htmlFor="grid-recipient">Email</Label>
+          <Input
```

Comment:
Wanna use our UI inputs here instead for consistency?

Reply:
yes! i am planning to do it, but in a different PR

Reply:
Any reason to _not_ include it here?

Reply:
oh, not at all, just to keep things separate :)

Reply:
just updated! with new UI